collector/database/chart.php

```diff
@@ -24,6 +24,7 @@
  * @copyright 2022, Catalyst IT
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
+define('MONTHSECS', WEEKSECS * 4);
```

Comment:
I'm not sure we should add this constant as it is misleading, I would prefer to see all uses of this removed instead, I've done this instead:

https://github.com/catalyst/moodle-tool_cloudmetrics/commit/7121d6e845459741e97a6f849c382b63a0a217bb